donate.html

```diff
@@ -312,47 +314,51 @@ <h5 class="margin-bottom-normal">{% filter ak_text:"donate_billing_information"
-                      </div>
-                      <div class="input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_address2">Billing address line 2 (optional)</label>
-                        <input id="id_address2" class="" type="text" name="address2">
-                      </div>
+                      {% if page.lang_or_en.iso_code == 'ja' %}
+                        {% include "./japanese_address_fields.html" %}
                       {% else %}
-                      <div class="input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_address1">{% filter ak_text:"field_billing_address" %}Billing
-                          address{% endfilter %}</label>
-                        <input id="id_address1" class="" type="text" name="address1">
-                      </div>
-                      {% endif %}
-                      <div class="input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_city">{% filter ak_text:"field_city" %}City{% endfilter %}</label>
-                        <input id="id_city" type="text" name="city">
-                      </div>
-                      <div class="ak-us-billing-fields input-select ak-input-type-user input-field-group ak-err-below">
-                        <input type="hidden" name="required" value="state">
-                        <label for="id_state">{% filter ak_text:"field_state" %}State{% endfilter %}</label>
-                        {% include "./state_select.html" %}
-                      </div>
-                      {% if page.allow_international %}
-                      <div class="ak-intl-billing-fields input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_region">{% filter ak_text:"field_region" %}Region{% endfilter %}</label>
-                        <input id="id_region" type="text" name="region" size="20">
-                      </div>
-                      {% endif %}
-                      <div class="ak-us-billing-fields input-text ak-input-type-user input-field-group ak-err-below">
-                        <input type="hidden" name="required" value="zip">
-                        <label for="id_zip">{% filter ak_text:"field_zip" %}Zip{% endfilter %}</label>
-                        <input id="id_zip" type="text" name="zip" maxlength="5" size="5">
-                      </div>
-                      {% if page.allow_international %}
-                      <div class="ak-intl-billing-fields input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_postal">{% filter ak_text:"field_postal" %}Postal Code{% endfilter %}</label>
-                        <input id="id_postal" type="text" name="postal" size="10">
-                      </div>
+                        {% if has_shippable_products %}

```

Comment:
seems like it would be appropriate for this "default" set of address fields to be extracted out into an include as well. but totally optional